diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3c178ab..b3595fb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,12 +45,8 @@ include_directories(.)
 include_directories(include/dualsynth)
 
 if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
-  if(CMAKE_SYSTEM_PROCESSOR MATCHES "AMD64")
-    set_source_files_properties(${SSE41_CODE_IMPL} PROPERTIES COMPILE_FLAGS "-march=penryn")
-  else()
-    set_source_files_properties(${SSE41_CODE_IMPL} PROPERTIES COMPILE_FLAGS "/arch:SSE2")
-  endif()
-
+  set_source_files_properties(${SSE41_CODE_IMPL} PROPERTIES COMPILE_FLAGS "/arch:SSE2")
+  
   if(CMAKE_GENERATOR_TOOLSET MATCHES "v[0-9]*_xp")
     target_compile_definitions(neo-f3kdb PRIVATE WINVER=0x502 _WIN32_WINNT=0x502)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Zc:threadSafeInit-")
@@ -70,12 +66,12 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "IntelLLVM")
 
 elseif(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang|AppleClang")
   if(CMAKE_SYSTEM_PROCESSOR MATCHES "arm" OR CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
-    target_compile_options(neo-f3kdb PRIVATE -O3 -funroll-loops)
-  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "AMD64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "i686")
-    target_compile_options(neo-f3kdb PRIVATE -O3 -funroll-loops)
+    target_compile_options(neo-f3kdb PRIVATE -Ofast -funroll-loops)
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "i686")
+    target_compile_options(neo-f3kdb PRIVATE -Ofast -march=native -funroll-loops)
     set_source_files_properties(${SSE41_CODE_IMPL} PROPERTIES COMPILE_FLAGS "-msse4.1")
   else()
-    target_compile_options(neo-f3kdb PRIVATE -O3 -funroll-loops)
+    target_compile_options(neo-f3kdb PRIVATE -Ofast -funroll-loops)
   endif()
 
 else()
@@ -98,4 +94,4 @@ endif()
 add_custom_command(
   TARGET neo-f3kdb POST_BUILD
   COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:neo-f3kdb> "../Release_${VERSION}/${_DIR}/$<TARGET_FILE_NAME:neo-f3kdb>"
-)
+)
\ No newline at end of file
diff --git a/src/version.hpp.in b/src/version.hpp.in
deleted file mode 100644
index 0bb98ba..0000000
--- a/src/version.hpp.in
+++ /dev/null
@@ -1,3 +0,0 @@
-#pragma once
-
-#define PLUGIN_VERSION "@VERSION@"
\ No newline at end of file
diff --git a/src/version.rc.in b/src/version.rc.in
deleted file mode 100644
index 159f480..0000000
--- a/src/version.rc.in
+++ /dev/null
@@ -1,30 +0,0 @@
-#include <winver.h>
-
-VS_VERSION_INFO VERSIONINFO
-FILEVERSION             1,0,0,0
-PRODUCTVERSION          1,0,0,0
-FILEFLAGSMASK           VS_FFI_FILEFLAGSMASK
-FILEFLAGS               0
-FILEOS                  VOS__WINDOWS32
-FILETYPE                VFT_DLL
-FILESUBTYPE             VFT2_UNKNOWN
-BEGIN
-    BLOCK "StringFileInfo"
-    BEGIN
-        BLOCK "040904b0"
-        BEGIN
-        VALUE "Comments",         "Neo-F3KDB"
-        VALUE "CompanyName",      ""
-        VALUE "FileDescription",  "Neo-F3KDB"
-        VALUE "FileVersion",      "@VERSION@"
-        VALUE "InternalName",     "Neo-F3KDB"
-        VALUE "OriginalFilename", "Neo-F3KDB.dll"
-        VALUE "ProductName",      "Neo-F3KDB"
-        VALUE "ProductVersion",   "@VERSION@"
-        END
-    END
-    BLOCK "VarFileInfo"
-    BEGIN
-        VALUE "Translation", 0x0409,1200
-    END
-END
